lib/ethereum/contract.rb

```diff
@@ -118,13 +118,13 @@ def send_transaction(tx_args)
         @client.eth_send_transaction(tx_args)["result"]
     end
 
-    def send_raw_transaction(payload, to = nil)
+    def send_raw_transaction(payload, to = nil, tx_args = nil)
       Eth.configure { |c| c.chain_id = @client.net_version["result"].to_i }
       @nonce ||= @client.get_nonce(key.address) - 1
       @nonce += 1
       args = {
         from: key.address,
-        value: 0,
+        value: ((!tx_args.nil? && tx_args.key?(:value) && !tx_args[:value].nil?) ? tx_args[:value] : 0),
```

Comment:
Instead of this:
`value: ((!tx_args.nil? && tx_args.key?(:value) && !tx_args[:value].nil?) ? tx_args[:value] : 0),`
we can use this:
`value: (tx_args.nil && tx_args[:value]) ? tx_args[:value] : 0),`

Could be even simpler if `tx_args = {}`:
`value: tx_args[:value] ? tx_args[:value] : 0,`